package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@babbel/stylelint-config",
+  "version": "1.0.0",
+  "author": "Eric L. Goldstein <egoldstein@babbel.com>",
+  "description": "Hierarchical Stylelint configuration collection that intends to be simple to use, layered, and shared with others",
+  "keywords": [
+    "stylelint",
+    "stylelint-config"
+  ],
+  "engines": {
+    "node": "^18.12.0 || ^20.9.0 || >=22.11.0"
```

Comment:
somewhere along the line Stylelint 16 upgraded their baseline Node version to `18.12.0`, so matching that here in addition to supporting all other current LTS versions